### PR TITLE
[15.0] [MIG] project_task_detail_report

### PR DESCRIPTION
**Pull request by Rajan-SCS**

### CODE REVIEW

File: project_task_detail_report/LICENSE/LICENSE [hunk 1]
```diff
@@ -0,0 +1,15 @@
+    OpenERP, Open Source Management Solution
+    Copyright (C) 2020-TODAY Serpent Consulting Services Pvt. Ltd. (<http://www.serpentcs.com>).
```
Comment (JayVora-SerpentCS) [inline project_task_detail_report/LICENSE/LICENSE]: 2022

**Comment by JayVora-SerpentCS** [comment]
@fetcheping please review

**Comment by JayVora-SerpentCS** [comment]
@tushar-serpentcs 
@fetcheping review pls

**Comment by tushar-serpentcs** [comment]
@JayVora-SerpentCS  PR Reviewd

**Pull request merged**